admin_guide/high_availability.adoc

```diff
@@ -0,0 +1,132 @@
+     and I want the cluster to automatically manage those resources.
+  2. As an administrator, I want my cluster to be assigned a set of virtual
+     IP addresses that the cluster manages and migrates (with zero or
+     minimal downtime) on failure conditions.
+  3. As an addendum to use case #2, the administrator should not be
+     required to perform any manual interaction to update the upstream
+     "discovery" sources (e.g. DNS). The cluster should service all the
+     assigned virtual IPs when atleast a single node is available - and
+     this should be inspite of the fact that the current available
+     resources are not sufficient to reach "critical mass" aka the
+     desired state.
+
+== ipfailover admin command
+The `oscadmin ipfailover` command helps to setup the VIP failover configuration. An administrator can configure IP failover on an entire cluster or as would normally be the case on a subset of nodes (as defined via a labelled selector). If you are running in production, it is recommended that the labelled selector for the nodes matches atleast 2 nodes to ensure you have failover protection and that you provide a --replicas=<n> value that matches the number of nodes for the given labelled selector.
+
+Running with the `--help` option will show you the supported options and their description. A small subset of these options pertinent to this document is described in the next section.
+
+****
+`$ oscadm ipfailover --help`
+****
+
+=== ipfailover command syntax
+
+        oscadm ipfailover [<name>] <options>
+        where:
+            <name> = Name of the IP failover configuration.
+                     Default: generated name (e.g.  ipfailover-1)
+            <options> = One or more options - see subset below.
+
+=== ipfailover command options (subset)
+The list of command options described here are a subset that are relevant to this document.
+
+            <options> = One or more of:
+                --create
+                --credentials=<credentials>
+                -l,--selector=<selector>
+                --virtual-ips=<ip-range>
+                -i|--interface=<interface>
+                -w|--watch-port=<port>
+
+            <credentials> = <string> - Path to .kubeconfig file containing credentials to use to contact the master.
+            <selector> = <string> - The node selector to use for running the HA sidecar pods.
+            <ip-range> = string - One or more comma separated IP address or ranges.
+                                  Example: 10.2.3.42,10.2.3.80-84,10.2.3.21
+            <interface> = <string> - The interface to use.
+                                     Default: Default interface on node or eth0
+            <port> = <number> - Port to watch for resource availability.
+                                Default: 80.
+            <string> = a string of characters.
+            <number> = a number ([0-9]*).
+
+
+== Setting up Routers with IP Failover
```

Comment:
There was an HA setup section in the architecture section of the routing doc that this can replace.  Since the ipfailover is generic you can describe all the features in a one doc and give all the router examples in the router doc.